### PR TITLE
add Contributing guide

### PR DESCRIPTION
**Pull request by ryanfoxtyler**
added initial contributing guide for repo and made a few tweaks to the Readme

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+
+**Modus SDKs** enable users to build apps in multiple languages, currently including Go and AssemblyScript.
+
+The **Modus CLI** drives the local development experience and is written in TypeScript.
+
+### Development environment
+
+The fastest path to setting up a development environment for Modus is through VS Code. The repo includes a set of configs to set VS Code up automatically.
+
+### Clone the Modus repository
```
Comment (mattjohnsonpint) [inline CONTRIBUTING.md:23]: Contributors will need to fork the repo and clone their fork.  Branching within their fork is then completely optional.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+```
+
+and create a branch for your work.
+
+```bash
+cd modus
+git checkout -b my-new-branch
+```
+
+### Building and running tests
```
Comment (mattjohnsonpint) [inline CONTRIBUTING.md:38]: This is OK for now, but we'll probably want to describe the monorepo structure and what projects can be built from where.  I can add that later if you want.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ryanfoxtyler) [reply CONTRIBUTING.md:23]: ah, right, addressed

File: CONTRIBUTING.md (hunk 2 above)
Comment (ryanfoxtyler) [reply CONTRIBUTING.md:27]: that would be great! agreed it can come later

**Review by mattjohnsonpint — APPROVED** [review]

**Pull request merged**